lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Show an error if too many warnings"
```

Comment:
Nonzero it is.

Re: wrapping: What I had before your comment (Show an error if too many warnings) only barely didn't wrap (after " - default: -1" was added), so I can't really add to that length. I'll also point out that at least in my console (80 chars), `--config` and `--ignore-pattern` currently wrap anyway, so it's not like there isn't precedent.

I can keep try to fiddle with it per your request, but I wonder if it might be better to just pick the best (i.e., most accurate without being too verbose) message we can now and open a separate issue for cleaning up the options later. (This would also allow us to group related options, sort anything that needs to be sorted, and decide what to do with the command line docs with regard to how closely they should follow the `options.js` config.)